src/components/expansion/ExpansionRuleForm.svelte

```diff
@@ -116,6 +136,21 @@
     }
   }
 
+  async function onTagsInputChange(event: TagsChangeEvent) {
+    const {
```

Comment:
Is there anyway to re-use the logic in this function across the different input handlers in the components?

Reply:
Not sure yet, some of the handlers do have different behavior since they only create tags but don't make associations (happens later on when the user actually submits the form).

Reply:
Yeah looks like it probably doesn't make a ton of sense.